CoEpi/native/NativeCore.swift

```diff
@@ -43,10 +43,48 @@ extension NativeCore: ServicesBootstrapper {
     // WARN: Must not run long operations. Currently executed synchronously during startup.
     func bootstrap(dbPath: String) -> Result<(), ServicesError> {
         let libResult: LibResult<ArbitraryType>? = bootstrap_core(dbPath)?.toLibResult()
+        NSLog("%@", "blah");
+        let x = register_callback{ (myInt, myBool, myCFString) in
+            if let cfStr: CFString = myCFString {
+                logToiOS(string: cfStr as String)
+            }
+        }
+        NSLog("register_callback returned : %d", x );
         return libResult?.toVoidResult().mapErrorToServicesError() ?? libraryFailure()
     }
 }
 
+func logToiOS(string: String){
+    let cs = ConsoleLog.init()
+    cs.d(string, tags: [.db])
+}
+
+func logToiOS(logMessage: CoreLogMessage){
+    let cs = ConsoleLog.init()
+    
+    guard let unmanagedText: Unmanaged<CFString> = logMessage.text else {
+        return
+    }
+    let text = unmanagedText.takeRetainedValue() as String
+    
+    switch logMessage.level {
+    case 0:
+        cs.v(text, tags: [LogTag.core])
+    case 1:
+        cs.d(text, tags: [LogTag.core])
+    case 2:
+        cs.i(text, tags: [LogTag.db])
```

Comment:
Why do we have different tags per level?

(btw, my tag enum probably should be plain strings, to allow core to set tags too. But we can let it like this for 0.3)

Reply:
Just tryin out if the logic works. 
Ok.